sechub-commons-model/src/main/java/com/mercedesbenz/sechub/commons/model/login/WebLoginTOTPConfiguration.java

```diff
@@ -0,0 +1,64 @@
+// SPDX-License-Identifier: MIT
+package com.mercedesbenz.sechub.commons.model.login;
+
+import javax.crypto.SealedObject;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.mercedesbenz.sechub.commons.core.security.CryptoAccess;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class WebLoginTOTPConfiguration {
+
+    public static final String PROPERTY_SEED = "seed";
+    public static final String PROPERTY_VALIDITY_IN_SECONDS = "validityInSeconds";
+    public static final String PROPERTY_TOKEN_LENGTH = "tokenLength";
+    public static final String PROPERTY_HASH_ALGORITHM = "hashAlgorithm";
+
+    private static final int DEFAULT_VALIDITY_IN_SECONDS = 30;
```

Comment:
We could make these default values public and use it inside the documentation (e.g. RESTDOC tests). If we would change the defaults at this point the documentation would always be valid.